lib/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -1718,6 +1745,13 @@ public synchronized void reset(long oldMsgSerial, ErrorInfo err) {
             startSerial = 0;
         }
 
+        public void resetStartSerial(int from) {
```

Comment:
Is this called now?

Reply:
Yes - this should be called from `addPendingMessagesToQueuedMessages`